k8s/gen/pt-nightly-fs-transformer-functional-v3-8.yaml

```diff
@@ -137,7 +143,7 @@
             - "name": "MODEL_DIR"
               "value": "gs://xl-ml-test-us-central1/k8s/fs-transformer/functional/v3-8/$(JOB_NAME)"
             - "name": "XLA_USE_BF16"
-              "value": "1"
```

Comment:
making sure you intended to turn off BF16 for both tests

Reply:
wait I can't read :)
Making sure you want to turn of BF16 for this test and turn it on for convergence test

Reply:
yes. intended.